koku/masu/test/processor/test_expired_data_remover.py

```diff
@@ -136,3 +139,86 @@ def test_remove_items_only(self, mock_purge):
         date = remover._calculate_expiration_date(line_items_only=True)
         remover.remove(line_items_only=True)
         mock_purge.assert_called_with(expired_date=date, simulate=False)
+
+    def test_delete_expired_cost_usage_report_manifest(self):
+        """
+        Test that expired CostUsageReportManifests are removed.
+
+        This test inserts CostUsageReportManifest objects,
+        And then deletes CostUsageReportManifest objects older than
+        the calculated expiration_date.
+        """
+
+        def insert_and_assert_delete_cost_usage_manifests(provider_type, provider_uuid):
+            """
+            Insert CostUsageReportManifests for a specific provider.
+
+            Assert that the expired records for that provider are deleted.
+            """
+            remover = ExpiredDataRemover(self.schema, provider_type)
+            expiration_date = remover._calculate_expiration_date()
+            this_month = datetime.today().replace(day=1)
+            day_before_cutoff = expiration_date - relativedelta.relativedelta(days=1)
+
+            # Record A
+            manifest_creation_datetime = this_month
+            manifest_updated_datetime = manifest_creation_datetime + relativedelta.relativedelta(days=2)
+            record_a_uuid = uuid4()
```

Comment:
These could be done in a for loop.

Reply:
@myersCody I changed it to be in a loop now.